code/modules/paperwork/paper.dm

```diff
@@ -539,6 +539,10 @@ var/global/list/paper_folding_results = list ( \
 	name = "paper- 'Suit Dispenser Manual - How to use them?'"
 	info = "Step 1: Place the items that you want the dispenser to dispense on top of one of them, preferably the one bellow this paper.<BR>\nStep 2: Click the dispenser, and choose <b>Define Preset from items on top</b>.<BR>\nStep 3: Click every dispenser you wish to see dispensing, and click <b>Choose a Preset</b>.<BR>\nTo re-use a dispenser, just click <b>Resupply</b>."
 
+/obj/item/weapon/paper/diy_soda
+	name = "paper- 'Instructions'"
+	info = "Thank you for purchasing Dr. Pecker's DIY Soda Kit!<BR>\nIt has been scientifically proven to bring your tastebuds into the delicious state and turn your teeth into a molar solution!<BR>\nNow as you may have guessed, you will have to mix this delicious beverage yourself.<BR>\nDon't worry, it's pretty basic stuff. Just remember to never lick the spoon!<BR>\nFirst, mix the contents of all three <b>small vials</b> into the <b>large flask</b>.<BR>\nThen, mix the contents of the <b>small flasks</b> into the <b>large flask</b>.<BR>\nAnd finally, get ready for our secret trademarked ingredient: <BR>\n<b>The element of surprise!</b>"
```

Comment:
please appreciate

Reply:
Post a pic of it in game please.

Reply:
![dreamseeker_2019-05-12_01-05-39](https://user-images.githubusercontent.com/40967382/57577643-17fb5b80-7452-11e9-9ecc-57d68bf282f7.png)